src/frontend/src/hooks/useForm.ts

```diff
@@ -1,43 +1,59 @@
-import { useState, useCallback, useMemo, useEffect } from "react";
+import { useState, useCallback, useMemo, useEffect } from 'react';
+const deepEqual = require('deep-equal');
```

Comment:
`import deepEqual from 'deep-equal'`?

Reply:
TypeScript wouldn't let me do that because of how the module is exported

Reply:
ahha